media-gfx/monica/monica-3.7.ebuild

```diff
@@ -17,14 +17,10 @@ IUSE=""
 DEPEND=">=x11-libs/fltk-1.1:1"
 RDEPEND="${DEPEND}
 	x11-apps/xgamma"
-
-src_prepare() {
-	epatch \
-		"${FILESDIR}"/${PN}-3.6-makefile-cleanup.patch \
-		"${FILESDIR}"/${P}-gcc44.patch
-
-	emake clean
-}
+PATCHES=(
+	"${FILESDIR}"/${PN}-3.6-makefile-cleanup.patch
+	"${FILESDIR}"/${P}-gcc44.patch
+)
```

Comment:
more to fix :smile: 
1. this is a stable ebuild. While `epatch` kinda sucks, modifying stable ebuilds is frowned upon. Revbump it to `-r1` to do your edits, leaving the original intact. Remember to make all arches unstable. You can use the `ebump` utility from `app-portage/gentoolkit-dev` for this.
2. you don't use any `eutils` functions anymore, hence nuke `eutils` from the `inherit` line
3. before `PATCHES`, define `DOCS=( authors ChangeLog news readme )`, and then do the following `src_install()`:
```
src_install() {
 	dobin monica
 	einstalldocs
}
```
And remember, you don't need to open a new PR for your edits. Also, after you added a new commit, always squash commits into one logical commit, see how `git rebase -i` is your friend: http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html